wemulate/ext/utils/restore.py

```diff
@@ -0,0 +1,45 @@
+import wemulate.ext.utils.common as common
+import wemulate.core.database.utils as dbutils
+import wemulate.utils.tcconfig as tcutils
+
+from wemulate.core.database.models import INCOMING, OUTGOING
+
+
+def restore_device() -> None:
+    """
+    Restores all connections and tcparameters stored in the database.
```

Comment:
Change to "_Restores all connections and tc parameters stored in the database._" for better readability.